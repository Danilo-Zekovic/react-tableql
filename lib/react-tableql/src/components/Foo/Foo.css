.foo {
  padding: 30px;
  background-color: tomato;
}
