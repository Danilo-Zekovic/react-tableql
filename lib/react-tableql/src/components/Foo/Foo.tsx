import React, { FC, useEffect } from 'react'

import logStuff from '../../utils/logStuff'

import './Foo.css'

interface FooProps {
  foo: string
}

const Foo: FC<FooProps> = ({ foo }) => {
  const halo = (): void => {
    logStuff('foo bar')
  }

  useEffect(() => {
    console.log('hi', foo)
  }, [foo])

  return (
    <div onClick={halo} className={`foo Foo`}>
      Hello World! {foo} is the best, but you knew that already.
    </div>
  )
}

export default Foo
