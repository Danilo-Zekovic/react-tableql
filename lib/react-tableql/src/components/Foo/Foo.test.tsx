import React from 'react'

import { render } from '@testing-library/react'

import Foo from './Foo'

describe('<SortArrows>', () => {
  it('default snapshot', () => {
    const { container } = render(<Foo foo="asd" />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
