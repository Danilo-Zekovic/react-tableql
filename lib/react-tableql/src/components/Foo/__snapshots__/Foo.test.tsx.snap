// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SortArrows> default snapshot 1`] = `
<div
  class="foo Foo"
>
  Hello World! 
  asd
   is the best, but you knew that already.
</div>
`;
